assets/cameras.csv

```diff
@@ -27,6 +27,8 @@ Mobotix AG,M15,10.18.168.61,16016,
 Mobotix AG,M15,10.7.46.153,16028,
 Mobotix AG,M15,10.20.245.115,16033,
 Mobotix AG,M15,10.20.210.120,,
+Mobotix AG,M16,10.26.62.41,12902,
+Mobotix AG,M16,TBA,12903,
```

Comment:
might be best to add this when the serial is known, it will need another PR in any case.

Reply:
@ozym, serial number now added